docs/tutorial-node-implementation.md

```diff
@@ -0,0 +1,494 @@
+
+
+### 2. Create a new Node Type
+
+Open Winery in your browser (default URL: http://localhost:8080) and click on the `Node Types` tab.
+Click on the `Add new` button on the right.
+
+![Create new node type in winery](./images/new_node_type/new.png)
+
+Enter the following values into the dialog and click on `Add`:
+
+| Field      | Value                                     | Explanation |
+|:-----------|:------------------------------------------|:------------|
+| Name       | CustomQiskit                              |             |
+| Versioning | Enabled                                   | (optional)  |
+| Version    | latest                                    | (optional)  |
+| Namespace  | `https://ust-quantil.github.io/nodetypes` |             |
+
+![New node type dialog](./images/new_node_type/new_dialog.png)
+
+Optionally create a README, choose a license and customize the appearance.
+
+![Node type readme](./images/new_node_type/readme.png)
+
+:information_source: In the case that an existing `Node Type` should be extended, go to the inheritance tab and select the `Node Type` to inherit from.
+
+
+### 3. Define Properties
+
+The original Qiskit `Node Type` does not have a `property`, but for this tutorial, we will define one. Open the `Property Definitions` tab.
+
+Select `Custom key/value pairs` to create new properties. Then click on the `Add` button to create a new Property.
+
+![Node type new property](./images/new_node_type/new_property.png)
+
+| Field | Value           | Explanation |
+|:------|:----------------|:------------|
+| Name  | `qiskitVersion` |             |
+| Type  | `xsd:string`    | data type   |
+
+Leave the rest empty and hit `Save`.
+
+![Node type new property dialog](./images/new_node_type/new_property_dialog.png)
+
+### 4. Create the Lifecycle Interface
+
+A `Node Type` in TOSCA needs to define `interfaces` that contain the `operations` that can be called/executed on that `Node Type`.
+
+First, open the `interfaces` tab.
+
+Click on the `Generate Lifecycle Interface` to create a new lifecycle interface for this `Node Type`.
+
+:information_source: The lifecycle interface defines operations for, e.g., installing, configuring, and starting a `Node Type`.
+
+![Generate Lifecycle Interface](./images/new_node_type/interfaces/generate_interfaces.png)
+
+For our Qiskit `Node Type`, we only need to install the dependency so we can delete all other operations of the lifecycle interface.
+
+![Delete unused operations](./images/new_node_type/interfaces/delete_operations.png)
+
+Click on the `install` operation to define input parameters. Then, click on the `Add` button of the input parameters.
+
+![Add input parameters](./images/new_node_type/interfaces/add_input_parameters.png)
+
+| Field    | Value           | Explanation                         |
+|:---------|:----------------|:------------------------------------|
+| Name     | `qiskitVersion` |                                     |
```

Comment:
Please add here a warning that the caseing MUST BE exactly as the required Property Definition.